### PR TITLE
Update Gherkin scenarios to match changes in release 0.11

### PR DESCRIPTION
**Pull request by radinamatic**
~WIP - don't merge yet!~ **Ready for review!**

### Summary

Reviewed and updated scenarios for changes in UI and polished wording here and there.

### Reviewer guidance
@mrpau-richard Long but nothing revolutionary... :wink: 
Review & merge when you have the chance!

### Contributor Checklist

- [x] Contributor has fully tested the PR manually
- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
~If there are any front-end changes, before/after screenshots are included~
~If this is an important user-facing change, PR or related issue has a 'changelog' label~

### Reviewer Checklist

- [ ] Automated test coverage is satisfactory
- [ ] Reviewer has fully tested the PR manually
- [ ] PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- [ ] External dependencies files were updated (`yarn` and `pip`)
- [ ] Documentation is updated
- [ ] Link to diff of internal dependency change is included
- [ ] CHANGELOG.rst is updated for high-level changes
- [ ] Contributor is in AUTHORS.rst

### CODE REVIEW

File: integration_testing/features/admin/admin-change-facility-settings.feature
Comment (indirectlylit) [inline integration_testing/features/admin/admin-change-facility-settings.feature:61]: Not critical for now, but just a note that these stories should probably have inverse scenarios. E.g. test state for both checked and unchecked.

**Review by indirectlylit — APPROVED** [review]
this looks awesome. I'll wait for @mrpau-richard to do final review and merge

File: integration_testing/features/admin/admin-change-facility-settings.feature
Comment (radinamatic) [reply integration_testing/features/admin/admin-change-facility-settings.feature:61]: Noted! 🙂 

I have several more points on the room-for-improvements list for other pickles, but wanted to jump on other tasks for now, and leave more scripture-rewrites for after the 0.11 release 😉

**Pull request merged**
